config/config.yml

```diff
@@ -58,6 +58,24 @@ mainnet:
         zil1c6akv8k6dqaac7ft8ezk5gr2jtxrewfw8hc27d: 1  # DUCK
         zil1r9dcsrya4ynuxnzaznu00e6hh3kpt7vhvzgva0: 1  # ZLF
         zil168qdlq4xsua6ac9hugzntqyasf8gs7aund882v: 1  # SRV
+    - name: "BLOX Liquidity Rewards"
+      reward_token_symbol: BLOX
+      reward_token_address_hex: "0x4268C34dA6Ad41a4cDeAa25cdEF6531Ed0c9a1A2"
+      distributor_name: BLOX
+      distributor_address_hex: "0x644590ca7de53c5275a1b5602099f3d65da4343f"
+      developer_address: zil1gy9v2fy4nsa5n5mx7grtt5sn205d6qduxkl84t
+      emission_info:
+        epoch_period: 604800 # 1 week
+        tokens_per_epoch: "1086900"
+        tokens_for_retroactive_distribution: "0"
+        retroactive_distribution_cutoff_time: 0
+        distribution_start_time: 1630483200
```

Comment:
This start time is 1 month ago, meaning 4 epochs will be generated?

Reply:
Ah that's not right, I've updated it to 1632902400

Reply:
Sorry, could you resolve the merge conflicts?

Reply:
For easy I've closed this pull request and created a fresh one over at
https://github.com/Switcheo/zap-api/pull/21